### PR TITLE
Ntd1hc/feat/robfw 328/update readme and document

### PR DESCRIPTION
**Pull request by ngoan1608**
Hi Thomas,

I have just updated the README files. Please review them first.

I am working with package's document and will add to this branch soon.

Thank you,
Ngoan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,87 +12,154 @@
```
Comment (test-fullautomation) [inline README.md:12]: Hi Ngoan,
the licence info must be removed here.
Thank you,
 Thomas

File: README.md [hunk 2]
```diff
@@ -12,87 +12,154 @@
 
 # TestResultWebApp (TRWebApp)
 
-::: {.contents local=""}
-:::
+## Table of Contents
+
+-   [Getting Started](#getting-started)
+    -   [How to install](#how-to-install)
```
Comment (test-fullautomation) [inline README.md]: Hi Ngoan,
'"how to install" is wrong here.
Thank you,
 Thomas

File: README.md [hunk 3]
```diff
@@ -12,87 +12,154 @@
...
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) was
+initially implemented at
+![IMG_BOSCH](packagedoc/additional_docs/pictures/Boschlogo.png) [Robert
+Bosch Car
+Multimedia](https://www.bosch.de/unser-unternehmen/bosch-in-deutschland/hildesheim/)
+and was open sourced in 2020.
 
-## Package Documentation
+### How to install
```
Comment (test-fullautomation) [inline README.md]: Hi Ngoan,
How to install is wrong here.
Somehow we should have at the end a small overview how to get the whole system run.
There was an architecture diagram as I remember. Some steps how to setup I have written in a readme. Can you please look if we can use this information. 
Best would be if this is part of  [TRWebApp's Documentation](https://github.com/test-fullautomation/testresultwebapp/blob/develop/TestResultWebApp/TestResultWebApp.pdf)
Thank you,
 Thomas

File: README.md [hunk 4]
```diff
@@ -12,87 +12,154 @@
 
 # TestResultWebApp (TRWebApp)
 
-::: {.contents local=""}
-:::
+## Table of Contents
+
+-   [Getting Started](#getting-started)
+    -   [How to install](#how-to-install)
+-   [Usage](#usage)
```
Comment (test-fullautomation) [inline README.md:18]: Hi Ngoan,
a chapter how to upload data is missing.
RobotLog2DB...
Thank you,
 Thomas

File: README.md [hunk 5]
```diff
@@ -12,87 +12,154 @@
...
+generic web based open source application which allows processing
+displaying of test result data in a sophisticated way.
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) uses
+a mysql data base, nodejs on server side and JavaScript on Web-Browser
+side.
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) was
+initially implemented at
+![IMG_BOSCH](packagedoc/additional_docs/pictures/Boschlogo.png) [Robert
```
Comment (test-fullautomation) [inline README.md]: Hi Ngoan,
I know, my first version was already not good :-)
The Bosch-Logo need to be aligned to the text. Please try to align it.
Thank you,
 Thomas

File: README.md [hunk 6]
```diff
@@ -12,87 +12,154 @@
...
+-   [About](#about)
+    -   [Maintainers](#maintainers)
+    -   [Contributors](#contributors)
+    -   [License](#license)
+
+## Getting Started
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) is a
+generic web based open source application which allows processing
+displaying of test result data in a sophisticated way.
```
Comment (test-fullautomation) [inline README.md:32]: ... which allows processing ***and*** displaying ...

File: README.rst [hunk 7]
```diff
@@ -13,108 +13,160 @@
...
+Contribution
+------------
+We are always searching support and you are cordially invited to help to improve 
+TRWebApp_ tool.
 
+Documentation
+-------------
+To understand more detail about the WebApp's features, the chart's meanings, 
+the displayed information on each view, please refer to 
+`TRWebApp’s Documentation`_.
```
Comment (test-fullautomation) [inline README.rst:151]: Hi Ngoan,
did you work on an update for this document, too?
I'm not sure, but I remember that you told this in our weekly sync.
Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Ngoan,
thank you for your pulll-request.
Please look into my findings.
Thank you,
 Thomas

**Comment by ngoan1608** [comment]
> Hi Ngoan, thank you for your pulll-request. Please look into my findings. Thank you, Thomas

Hi Thomas,

I have just updated some parts of package document and would like you to give your feeling about them.
There are the introduction section and document for datable view are under documenting.

Please review  the initial https://github.com/test-fullautomation/testresultwebapp/blob/ntd1hc/feat/ROBFW-328/update-readme-and-document/TestResultWebApp/TestResultWebApp.pdf file and let me know if it need to be improved.
I am trying to consolidate all the features and how to use them.

P/s: the missing sections and your findings will be fixed in next commits.

Thank you,
Ngoan

**Comment by test-fullautomation** [comment]
Hi Ngoan,

very good! I looked into [TestResultWebApp.pdf](https://github.com/test-fullautomation/testresultwebapp/blob/ntd1hc/feat/ROBFW-328/update-readme-and-document/TestResultWebApp/TestResultWebApp.pdf).
It has small typos, but is already a very good starting point for somebody who want to use it.

At the beginning we should have a friendly hint, that somebody who want to use our WebApp need to have some knowledge about Web- and Database Technologies. 

Thank you,
 Thomas

**Comment by test-fullautomation** [comment]
Hi Ngoan,
your input was very good!
I made some corrections and extensions and merge now.
Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]

**Pull request merged**